lib/svg-sprite.js

```diff
@@ -84,15 +83,16 @@ SVGSpriter.prototype._namespacePow = [];
 SVGSpriter.prototype.add = function(file, name, svg) {
     // If no vinyl file object has been given
     if (!this._isVinylFile(file)) {
-        file = trim(file);
+        file = file.trim();
         let error = null;
 
         // If the name part of the file path is absolute
         if (name && path.isAbsolute(name)) {
             error = format('SVGSpriter.add: "%s" is not a valid relative file name', name);
         } else {
-            name = trimStart(trim(name), `${path.sep}.`) || path.basename(file);
-            svg = trim(svg);
+            name = trimStart(name.trim(), `${path.sep}.`) || path.basename(file);
+            // TODO: Avoid Buffer -> String -> Buffer conversion of svg.
+            svg = svg.toString().trim();
```

Comment:
Just wondering, would it make more sense to check if `svg` is Buffer with `Buffer.isBuffer()` and if so only then call `toString()`?

Reply:
This would seem a little overengineering for me.